docs/guides/akamai/new-dbaas-partner/index.md

```diff
@@ -0,0 +1,106 @@
+---
+slug: new-dbaas-partner
+title: "New Database Service Partner Update"
+description: "Akamai partners with Aiven for more robust managed database services."
+og_description: "Akamai partners with Aiven for more robust managed database services."
+authors: ["Akamai"]
+contributors: ["Akamai"]
+published: 2024-11-05
+keywords: ['dbaas','dedicated','migration']
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+_build:
+    list: false
+noindex: true
+---
+
+## New Managed Database Services powered by Aiven
+
+Beginning November 14, 2024, Akamai will re-introduce its managed database service in partnership with Aiven to bring customers higher performance, reliability, and flexibility. At launch, the new service is available in 20 core compute sites across the globe for distribution and low latency. New sites will be added as we move forward.
+
+With the re-introduction of this new service, the resources currently available will be retired. Please study the guides below about how to create new databases, migrate existing databases to the new infrastructure, and how the pricing changes.
+
+## New database creation
+Starting November 14th, 2024, you can set up new database instances through the Linode API and in Cloud Manager. These locations might be different from where customers currently have running databases.
+
+The list of available locations are:
+
+**NA**
+- Atlanta
+- Chicago
+- Dallas
+- Miami
+- Fremont
+- Newark
+- Toronto
+- Washington
+
+**EMEA**
+- Frankfurt
+- London
+- Paris
+- Amsterdam
+- Stockholm
+
+**APAC**
+- Chennai
+- Melbourne
+- Mumbai
+- Singapore
+- Sydney
+- Osaka
+- Tokyo
+
+## Migration of existing databases
+We highly recommend migrating data from the old database infrastructure as soon as possible because, from November 14th, 2024, the old database infrastructure won’t receive updates anymore.
+
+All customers will be offered the ability to select the final migration schedule window. There will be an option to choose one or more database instances for each migration process. **If a customer does not initiate the migration after two reminders, we will initiate the migration from the backend during off-business hours.**
```

Comment:
Same note for the second sentence here - wonder if it should read "If you do not initiate the migration..."